src/app/service/constants/constants.ts

```diff
@@ -20,7 +21,7 @@ export default Object.freeze({
 	DB_COLLECTION_NEXT_AUTH_SESSIONS: 'sessions',
 	DB_COLLECTION_NEXT_AUTH_ACCOUNTS: 'accounts',
 
-	MONGODB_URI_PARTIAL: `${process.env.MONGODB_PREFIX}://${process.env.MONGODB_USERNAME}:${process.env.MONGODB_PASS}@${process.env.MONGODB_CLUSTER}/`,
+	MONGODB_URI_PARTIAL: `${process.env.MONGODB_PREFIX}://${process.env.MONGODB_USERNAME}:${process.env.MONGODB_PASS}@${process.env.MONGODB_CLUSTER}`,
```

Comment:
Please avoid making changes to this line. If there is trouble in connecting to the MongoDB instance, the best approach will be to utilize Docker to fire up your test bot. Altering this line will alter Serendipity's and DEGEN's ability to connect to their respective db tooling.